tests/foreman/cli/test_errata.py

```diff
@@ -1390,9 +1448,116 @@ def test_positive_user_permission(self):
         @Assert: Check that the new user is able to see errata for one
         product only.
 
-        @caseautomation: notautomated
-
         """
+        user_password = gen_alphanumeric()
+        user_name = gen_alphanumeric()
+        org = self.org3
+        product1 = self.org3_product1
+        product2 = self.org3_product2
+        # get the available permissions
+        permissions = Filter.available_permissions()
+        user_required_permissions_names = ['view_products']
+        # get the user required permissions ids
+        user_required_permissions_ids = [
+            permission['id']
+            for permission in permissions
+            if permission['name'] in user_required_permissions_names
+        ]
+        self.assertGreater(user_required_permissions_ids, 0)
+        # create a role
+        role = make_role()
+        # create a filter with the required permissions for role with product
+        # one only
+        make_filter({
+            'organization-ids': org['id'],
+            'permission-ids': user_required_permissions_ids,
+            'role-id': role['id'],
+            'search': 'name = {0}'.format(product1['name'])
+        })
+        # create a new user and assign him the created role permissions
+        user = make_user({
+            'admin': False,
+            'login': user_name,
+            'password': user_password
+        })
+        User.add_role({'id': user['id'], 'role-id': role['id']})
+        # make sure the user is not admin and has only the permissions assigned
+        user = User.info({'id': user['id']})
+        self.assertEqual(user['admin'], 'no')
+        self.assertEqual(set(user['roles']), {DEFAULT_ROLE, role['name']})
+        # try to get organization info
+        # get the info as admin user first
+        org_info = Org.info({'id': org['id']})
+        self.assertEqual(org['id'], org_info['id'])
+        self.assertEqual(org['name'], org_info['name'])
+        # get the organization info as the created user
+        with self.assertRaises(CLIReturnCodeError) as context:
+            Org.with_user(user_name, user_password).info({'id': org['id']})
+        self.assertIn(
+            'Forbidden - server refused to process the request',
+            context.exception.stderr
+        )
+        # try to get the erratum products list as the created user
```

Comment:
@oshtaier,  no we have have had no requirements for that in the doc string, I think that this is a missing Assert in the doc (I think that it a good idea to check that too as it cost nothing to do it, as all the resources are already allocated) and there no other permissions tests, perhaps I should update the doc instead?.